tests/BuildSystem/Build/basic.llbuild

```diff
@@ -66,7 +66,7 @@ commands:
   "<env-1>":
     tool: shell
     outputs: ["<env-1>"]
-    args: ["/usr/bin/env"]
+    args: /usr/bin/env | /usr/bin/sort
```

Comment:
Why did this change?

Reply:
(I mean, it looks like a good change just curious)

Reply:
Because of how I split the layers, the LLBUILD_LANE_ID ends up at the top of the environment variable list now. 

Specifically, the LaneBasedExecutionQueue puts LLBUILD_LANE_ID into it first, then user variables, then the spawnProcess code, which handles the control channel adds LLBUILD_CONTROL_FD and LLBUILD_TASK_ID last.